### PR TITLE
rework intro page

### PR DESCRIPTION
**Pull request by moredip**
I sort of took a hatchet to the original content. Not sure if we want this new content as a replacement or as a separate section. I'm happy for you to do what you will with it.

### CODE REVIEW

**Comment by beeme1mr** [comment]
Could you make the "flag eval" section of the images a bit larger? Also, The `typical feature flag system` image doesn't include evaluation context but I believe it should.

**Comment by beeme1mr** [comment]
I would like to update the architecture diagram used on the provider page with what you've created. Could you please move it to a less nested folder?

File: docs/reference/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Welcome to OpenFeature!
+
+OpenFeature is an open standard for feature flagging, created to support a robust feature flag ecosystem using cloud native technologies.
+
+## What are feature flags?
+
+In the most basic case, you can think of a feature flag as an if/else statement that can be controlled at runtime. Feature flags allow application behavior to be altered without the deployment of new code.
+
+This capability serves a variety of purposes. You can reduce the need for long-running feature branches. You can hide work-in-progress functionality from end users, while still exposing it for internal testing. You can perform canary releases - rolling out a new feature to an initially small subset of users. You can perform A/B testing. You can safely degrade parts of a production system which are experiencing an outage. You can restrict access to specific product functionality - an admin page, for example, or premium features that are only exposed to paid subscribers.
```
Comment (benjiro) [inline docs/reference/intro.mdx:15]: Correct pronoun usage
```suggestion
This capability serves a variety of purposes. You can reduce the need for long-running feature branches. You can hide work-in-progress functionality from end users, while still exposing it for internal testing. You can perform canary releases - rolling out a new feature to an initially small subset of users. You can perform A/B testing. You can safely degrade parts of a production system that are experiencing an outage. You can restrict access to specific product functionality - an admin page, for example, or premium features that are only exposed to paid subscribers.
```

File: docs/reference/intro.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Given all these requirements (along with others such as an admin UI, audit trails, environment management, and more) it's clear that full use of feature flags requires a feature flagging system - typically a stand-alone feature-flagging service along with a client library which interacts with that service.
+
+<figure>
+<img src={require('@site/static/img/docs/reference/intro/ff-service.png').default}/>
+<figcaption><i>A typical feature-flagging system</i></figcaption>
+</figure>
+
+## What is OpenFeature?
+
+OpenFeature provides a shared, standardized feature flagging client - an *SDK* - which can be plugged in to various 3rd-party feature flagging *providers*. Whether you're using an open-source system or a commercial product, whether it's self-hosted or cloud-hosted, OpenFeature provides a consistent, unified API for developers to use feature flagging in their applications.
```
Comment (benjiro) [inline docs/reference/intro.mdx:28]: one word?
```suggestion
OpenFeature provides a shared, standardized feature flagging client - an *SDK* - which can be plugged into various 3rd-party feature flagging *providers*. Whether you're using an open-source system or a commercial product, whether it's self-hosted or cloud-hosted, OpenFeature provides a consistent, unified API for developers to use feature flagging in their applications. 
```

File: docs/reference/intro.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+### Providers
+
+[Providers](./concepts/02-provider.mdx) are the "translation layer" between the evaluation API and the flag management system in use. Providers are responsible for mapping the arguments supplied to the evaluation API to their equivalent representation in the associated flag management system. Providers might wrap a vendor SDK, call a bespoke flag evaluation REST API, or even parse some locally stored file to resolve flag values.
+
+### Hooks
+
+[Hooks](./concepts/04-hooks.mdx) are a mechanism that allow for the addition of arbitrary behavior at various points in the _flag evaluation life-cycle_. Hooks let you extend the OpenFeature SDK, adding functionality such as validating a resolved flag value, modifying or adding data to the evaluation context, logging, telemetry, and tracking.
```
Comment (benjiro) [inline docs/reference/intro.mdx:51]:
```suggestion
[Hooks](./concepts/04-hooks.mdx) are a mechanism that allows for the addition of arbitrary behavior at various points in the _flag evaluation life-cycle_. Hooks let you extend the OpenFeature SDK, adding functionality such as validating a resolved flag value, modifying or adding data to the evaluation context, logging, telemetry, and tracking.
```

**Review by benjiro — COMMENTED** [review]
A couple of comments around grammar. 

Awesome job enhancing the introduction page love it 🏆

File: docs/reference/intro.mdx (hunk 3 above)
Comment (beeme1mr) [reply docs/reference/intro.mdx:51]: @moredip this is a great change that I would like to get in as soon as possible. When you have a moment, would you be able to address the comments above?

**Comment by moredip** [comment]
> Could you make the "flag eval" section of the images a bit larger? Also, The `typical feature flag system` image doesn't include evaluation context but I believe it should.

Yep, I can make the ‘flag eval’ text a bit bigger.

I intentionally left some detail out of the first diagram in the hopes of making it a less busy/overwhelming.

**Comment by moredip** [comment]
> I would like to update the architecture diagram used on the provider page with what you've created. Could you please move it to a less nested folder?

Yep happy to. Do you have a specific location that you think would be best?

**Comment by beeme1mr** [comment]
> Yep happy to. Do you have a specific location that you think would be best?

I think moving it up one folder would work.

**Comment by moredip** [comment]
@beeme1mr I made the "flag eval" part a bit bigger, and moved the images. LMK if you need anything else!

**Review by benjiro — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]
Looks good! Thanks for putting this together.

**Pull request merged**